frontend/pages/registration.js

```diff
@@ -0,0 +1,8 @@
+import Link from 'next/link'
+
+import Layout from '../components/Layout'
+import RegistrationForm from '../components/Registration'
+
+export default function Registration() {
+  return <RegistrationForm />
```

Comment:
Right now this component isnt really doing much do you think it will do anything in the future? If not you could just delete the component and paste the code in here.

Reply:
Hmm I know what you mean but I do like having them separate, once its more final then that'll make more sense to do.